core/src/main/java/nl/nn/adapterframework/pipes/RhinoPipe.java

```diff
@@ -42,21 +42,8 @@
  * This pipe takes all input and pushes it into javascript runtime.
  * The invoke method is called to initialize the runtime
  * Afterward the results are evaluated.
- * <table border="1">
- * <p>
- * <b>Parameters:</b>
- * <tr>
- * <th>name</th>
- * <th>type</th>
- * <th>remarks</th>
- * </tr>
- * <tr>
- * <td><i>any</i></td>
- * <td><i>any</i></td>
- * <td>Any parameters defined on the pipe will be Concatenated into one string and added to input</td>
- * </tr>
- * </table>
- * </p>
+ * 
+ * @ff.parameters Any parameters defined on the pipe will be Concatenated into one string and added to input
```

Comment:
ik snap dat het gecopy paste is, maar ik denk toch dat Concatenated met een kleine c moet..